cmd/ftl/cmd_serve.go

```diff
@@ -142,9 +142,11 @@ func (s *serveCmd) run(ctx context.Context, projConfig projectconfig.Config, ini
 	}
 
 	// Wait for controller to start, then run startup commands.
-	if err := waitForControllerOnline(ctx, time.Second*10, client); err != nil {
```

Comment:
Ah yep, almost certainly the issue.

Reply:
Maybe we should make this a flag?

Reply:
It totally relies on the computer speed and complexity of the project, so sure. Will do.